docs/api-docs/catalog/products-overview.md

```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
 }
 ```
 
+## Product Sort Order (Beta)
+
+[Product Sort Order](add-link-here) allows you to manage the sort order of products displayed on any given category page. Products assigned to multiple storefront categories can have different sort order values per category.
+
+### Product sorting on a storefront 
+
+The Catalog API supports two manually managed methods of product sorting: on a category level and on a product level. If a user combines both sorting methods on a storefront, products are sorted based on the following sorting method priority:
+
+1. Manually specified sort order on a category level.
+2. Manually specified sort order on a product level. `0` by default. 
+
+If there is no sort order value on a category level, products are sorted by values on a product level.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+
+### Note
+> Products with the same sort order value either on a category or a product level are sorted by `product id` as a second criterion.

```

Comment:
The Catalog API sorts by `product id` as a second criterion for products with the same sort order value either on a category or a product level.